src/gui/mzroll/ligandwidget.cpp

```diff
@@ -394,6 +392,53 @@ void LigandWidget::showTable() {
     treeWidget->setSortingEnabled(true);
 }
 
+void LigandWidget::markAsDone(Compound* compound) {
```

Comment:
do anything inside the function if  the following is true  
`if (compound != nullptr)` 

same check for  `QTreeWidgetItem* item` just for more readability